recipes/avrocpp/1.8.2/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha1: {{ hash }}
 
 build:
-  number: 4
+  number: 5
 
```

Comment:
I cannot understand what this variable refers to

Reply:
The is the Conda build number, which must be incremented each time there is a change to the recipe that requires the package to be re-deployed (see Conda docs for `meta.yaml`).